src/lib.rs

```diff
@@ -60,6 +60,8 @@ pub mod binding;
 pub mod event;
 pub mod message;
 
+pub mod test_data;
```

Comment:
Would this work? If so, it would obviate adding a special feature, I think.

```suggestion
#[cfg(test)]
pub mod test_data;
```

Reply:
I tested it and it didn't work. I added `test` feature that defines this and there's an example of how to use it in `actix-web-example`